### PR TITLE
Jakarta Activation dependency update (2.0.2-SNAPSHOT).

### PR DESCRIPTION
**Pull request by Tomas-Kraus**

### CODE REVIEW

File: jaxb-ri/jxc/pom.xml [hunk 1]
```diff
@@ -113,7 +113,7 @@
                         <configuration>
                             <outputDirectory>${project.build.directory}/schemagen-cp</outputDirectory>
                             <includeArtifactIds>
-                                jakarta.activation,jakarta.xml.bind-api,
+                                jakarta.activation-api,jakarta.xml.bind-api,
```
Comment (lukasj) [inline jaxb-ri/jxc/pom.xml:116]: angus-activation should probably be added here

File: jaxb-ri/xjc/pom.xml [hunk 2]
```diff
@@ -145,7 +145,7 @@
                         <configuration>
                             <outputDirectory>${project.build.directory}/xjctask-cp</outputDirectory>
                             <includeArtifactIds>
-                                jakarta.activation,istack-commons-tools,istack-commons-runtime,
+                                jakarta.activation-api,istack-commons-tools,istack-commons-runtime,
```
Comment (lukasj) [inline jaxb-ri/xjc/pom.xml:148]: angus-activation should probably be added here

File: jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/JAXBContextImpl.java [hunk 3]
```diff
@@ -716,10 +716,6 @@ public UnmarshallerImpl createUnmarshaller() {
         return new UnmarshallerImpl(this,null);
     }
 
-    public Validator createValidator() {
-        throw new UnsupportedOperationException(Messages.NOT_IMPLEMENTED_IN_2_0.format());
```
Comment (lukasj) [inline jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/JAXBContextImpl.java:720]: if this message is no longer used, remove it also from properties file

File: jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/unmarshaller/UnmarshallerImpl.java [hunk 4]
```diff
@@ -499,29 +499,27 @@ public void setAttachmentUnmarshaller(AttachmentUnmarshaller au) {
     /**
      * @deprecated since 2.0
      */
-    @Override
```
Comment (lukasj) [inline jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/unmarshaller/UnmarshallerImpl.java:502]: add Deprecated annotation here (and other places you're modifying if it already contains doresponding javadoc tag like it is in this case)

File: jaxb-ri/jxc/pom.xml (hunk 1 above)
Comment (Tomas-Kraus) [reply jaxb-ri/jxc/pom.xml]: Added.

File: jaxb-ri/xjc/pom.xml (hunk 2 above)
Comment (Tomas-Kraus) [reply jaxb-ri/xjc/pom.xml]: Added.

File: jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/JAXBContextImpl.java (hunk 3 above)
Comment (Tomas-Kraus) [reply jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/JAXBContextImpl.java:720]: Messages.NOT_IMPLEMENTED_IN_2_0 deleted.

File: jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/unmarshaller/UnmarshallerImpl.java (hunk 4 above)
Comment (Tomas-Kraus) [reply jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/unmarshaller/UnmarshallerImpl.java:502]: Added to both deprecated methods.

File: jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/Messages.java [hunk 5]
```diff
@@ -20,7 +20,6 @@
     ILLEGAL_PARAMETER, // 2 args
     UNABLE_TO_FIND_CONVERSION_METHOD, // 3 args
     MISSING_ID, // 1 arg
-    NOT_IMPLEMENTED_IN_2_0,
```
Comment (lukasj) [inline jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/Messages.java:23]: also from corresponding `Messages.properties` file (in all language variants)

**Review by lukasj — APPROVED** [review]
LGTM

**Pull request merged**